src/bodywork/k8s/secrets.py

```diff
@@ -143,6 +143,25 @@ def secret_exists(
         return False
 
 
+def secret_group_exists(namespace: str, group: str) -> bool:
+    """ Does the specified secret group exist
```

Comment:
Doctoring format needs adjusting so that there isn't a blank space before the start of the first line (needs a full-stop, too). And then the items below shouldn't be indented.

Reply:
Good spot, my IDE used to generate these automatically but it's not doing it after I got my laptop repaired for some reason.